uw_sws/section.py

```diff
@@ -190,8 +192,34 @@ def _get_sections_by_person_and_term(person,
         params.append(("delete_flag", ','.join(sorted(delete_flag)),))
 
     url = "%s?%s" % (section_res_url_prefix, urlencode(params))
+    return get_resource(url)
```

Comment:
Why did we remove the sectionref conversion here?

Reply:
The new function get_last_section_by_instructor_and_terms wants to modify the returned json before models are built, as an optimization.